src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -320,19 +320,45 @@ public boolean conferenceRequest(
         room = room.toLowerCase();
 
         JitsiMeetConferenceImpl conference;
+        String id;
         synchronized (conferencesSyncRoot)
         {
-            if (!conferences.containsKey(room))
+            conference = conferences.get(room);
+            if (conference == null)
+            {
+                id = generateConferenceId();
```

Comment:
it seems like it would be even better for `createConference` to do all this work, actually, but we don't need to do that in this PR, perhaps note as a FIXME?